src/containers/Home/homeReducer.js

```diff
@@ -0,0 +1,151 @@
+const initialState = {
+    reminders:[], 
+    error:null, 
+}
+
+
+export default function homeReducer (state = initialState , action ){
+   switch(action.type){
+        case "FETCH_REMINDER_PENDING": {
+            return {
+                ...state,
+                fetching:true
+            }
+        }
+        case "FETCH_REMINDER_REJECTED": {
+            return {
+                ...state,
+                fetching:false,
+                error:action.payload
+            }
+        }
+        case "FETCH_REMINDER_FULFILLED": {
+            return {
+                ...state,
+                fetching:false,
+                reminders:action.payload.data
+            }
+        }
+        case "SEND_REMINDER_PENDING": {
+            return {
+                ...state,
+                fetching:true
+            }
+        }
+        case "SEND_REMINDER_REJECTED": {
+            alert('you cannot select previous date and time ');
```

Comment:
Message message like this wrong!, because server will throw different errors based on the input. But, the way you did will always shows the message `you cannot select previous date and time` regardless of what is exact error. 

For example: Server might throw error, 'phone number is not valid' - but you always show the message `you cannot select previous date and time` - which is not correct!